docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,98 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant additional two years of 
+extend security coverage. This service is only available on Trusty, since the release has reached
+its end of life support for esm-infra.
+
+.. Make sure Pro is up to date
+.. include:: ./enable-disable/update-pro.txt
+
+Check the status of the services
+================================
+
+After you have :ref:`attached your subscription <get_token_and_attach>` and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    ua status
+
+This will show you which services are enabled or disabled on your machine
+(output truncated for brevity):
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+If you are entitled to the service, you can simply enable it by running:
+
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What to do if esm-infra-legacy is not entitled
+===============================================
+
+If status is showing that you are not entitled to the service, you
+will need to buy access to the service through the `Pro webpage`_.
```

Comment:
Is that the recommended way? Or should we say "contact sales/support"?

Reply:
Good question, I am assuming we can do this through the portal, but maybe it is safer to say for the user to look for support